docs/roadmap.md

```diff
@@ -0,0 +1,30 @@
+# Roadmap
+
+_Complementary to Phase 2 Scope: #13113_
+
+This document outlines some of the features currently in development or being considered for the project. It should not be confused with the product roadmap for WordPress itself, even if some areas naturally overlap. The main purpose of it is to give visibility to some of the key problems remaining to be solved and as an invitation for those wanting to collaborate on some of the more complex issues to learn about what needs help.
+
+Gutenberg is already in use by millions of sites through WordPress, so in order to make substantial changes to the design or updating specifications it is advisable to consider a discussion process ("Request for Comments") showing both an understanding of the impact, both positives and negatives, trade offs and opportunities.
+
+## Projects
+
+- **Block Registry** — define an entry point for block identification. (See active RFC at #13693.)
+- **Live Component Library** — a place to visualize and interact with the UI components and block tools included in the packages.
+- **Modular Editor** — allow loading the block editor in several contexts without a dependency to a post object. (Ongoing; pending tasks at #14043.)
+- **Better Validation** — continue to refine the mechanisms used in validating editor content. (See in depth overview at #11440 and #7604.)
+- **Block Areas** — build support for areas of blocks that fall outside the content (including relationship with templates, registration, storage, and so on). (See overview at #13489.)
+- **Multi-Block Editing** — allow modifying attributes of multiple blocks of the same kind at once.
+- **Rich Text Roadmap** — continue to develop the capabilities of the rich text package. (See overview at #13778.)
+- **Common Block Functionality** — coalesce into a preferred mechanism for creating and sharing chunks of functionality (block alignment, color tools, etc) across blocks with a simple and intuitive code interface. (Suggested exploration: React Hooks.)
+- **Responsive Images** — propose mechanisms for handling flexible image sources that can be optimized for loading and takes into account their placement on a page (within main content, a column, sidebar, etc).
+- **Async Loading** — propose a strategy for loading block code only when necessary in the editor without overhead for the developer or disrupting the user experience.
+- **Styles** — continue to develop the mechanisms for managing block style variations and other styling solutions. (See overview at #7551 and #9534.)
+- **Bundling Front-end Assets** — explore ways in which front-end styles for blocks could be assembled based on which blocks are used in a given page response. (See overview at #5445.)
```

Comment:
Another good thing that needs improvements and that is technically challenging is the "Transforms API". Related issue here https://github.com/WordPress/gutenberg/issues/14755

Reply:
Let's add it.

Reply:
done